src/BannedSymbols.txt

```diff
@@ -17,11 +17,21 @@ M:System.Threading.Tasks.Task`1.ConfigureAwait(System.Boolean); "ConfigureAwait(
-M:System.Threading.Tasks.Task.Run`1(System.Func{System.Threading.Tasks.Task{``0}},System.Threading.CancellationToken); Task.Run should be replaced with IProjectThreadingService.Fork, which prevents the project being unloaded while the Task is running
\ No newline at end of file
+M:System.Threading.Tasks.Task.Run(System.Action); Task.Run should be replaced with IProjectThreadingService.Fork2, which prevents the project being unloaded while the Task is running and ensures that failed tasks are reported as non-fatal watsons
+M:System.Threading.Tasks.Task.Run`1(System.Func{``0}); Task.Run should be replaced with IProjectThreadingService.Fork2, which prevents the project being unloaded while the Task is running and ensures that failed tasks are reported as non-fatal watsons
+M:System.Threading.Tasks.Task.Run`1(System.Func{System.Threading.Tasks.Task{``0}}); Task.Run should be replaced with IProjectThreadingService.Fork2, which prevents the project being unloaded while the Task is running and ensures that failed tasks are reported as non-fatal watsons
+M:System.Threading.Tasks.Task.Run(System.Func{System.Threading.Tasks.Task}); Task.Run should be replaced with IProjectThreadingService.Fork2, which prevents the project being unloaded while the Task is running and ensures that failed tasks are reported as non-fatal watsons
+M:System.Threading.Tasks.Task.Run(System.Action,System.Threading.CancellationToken); Task.Run should be replaced with IProjectThreadingService.Fork2, which prevents the project being unloaded while the Task is running and ensures that failed tasks are reported as non-fatal watsons
+M:System.Threading.Tasks.Task.Run`1(System.Func`1{``0},System.Threading.CancellationToken); Task.Run should be replaced with IProjectThreadingService.Fork2, which prevents the project being unloaded while the Task is running and ensures that failed tasks are reported as non-fatal watsons
+M:System.Threading.Tasks.Task.Run(System.Func{System.Threading.Tasks.Task},System.Threading.CancellationToken); Task.Run should be replaced with IProjectThreadingService.Fork2, which prevents the project being unloaded while the Task is running and ensures that failed tasks are reported as non-fatal watsons
+M:System.Threading.Tasks.Task.Run`1(System.Func{System.Threading.Tasks.Task{``0}},System.Threading.CancellationToken); Task.Run should be replaced with IProjectThreadingService.Fork2, which prevents the project being unloaded while the Task is running and ensures that failed tasks are reported as non-fatal watsons
+
+M:Microsoft.Threading.Tasks.TplExtensions.Forgot(System.Threading.Tasks.Task); Use IProjectFaultHandlerService.RegisterFaultHandler2 to ensure that failed tasks are reported as non-fatal watsons and assigned to the correct component
```

Comment:
This is Microsoft.VisualStudio.Threading.TplExtensions.

Reply:
```suggestion
M:Microsoft.Threading.Tasks.TplExtensions.Forget(System.Threading.Tasks.Task); Use IProjectFaultHandlerService.RegisterFaultHandler2 to ensure that failed tasks are reported as non-fatal watsons and assigned to the correct component
```

Reply:
i.e. it's Forget, not Forgot